app/controllers/api/v1/user_events_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Api::V1::UserEventsController < ApplicationController
+    before_action :information, only: [:create, :remove_event_type, :inc_event_type, :inc_name]
```

Comment:
Added before action to get the information needed to make these helper methods in the `private` section function without repeating myself.